tests/unit_tests/shared/share/test_ecl_run.py

```diff
@@ -92,14 +89,14 @@ def init_ecl100_config(monkeypatch, tmpdir):
         },
     }
     with tmpdir.as_cwd():
-        with open("ecl100_config.yml", "w", encoding="utf-8") as f:
-            f.write(yaml.dump(conf))
+        with open("ecl100_config.yml", "w", encoding="utf-8") as filehandle:
+            filehandle.write(yaml.dump(conf))
         monkeypatch.setenv("ECL100_SITE_CONFIG", "ecl100_config.yml")
         yield
 
 
-@pytest.fixture
-def init_flow_config(monkeypatch, tmpdir):
+@pytest.fixture(name="init_flow_config")
+def fixture_init_flow_config(monkeypatch, tmpdir):
```

Comment:
Why `name=init_flow_config` needs to be there? I could not find it quickly on a search...

Reply:
It is `pylint` that gives a warning on `redefined-outer-name` prior to this PR. The solution to that pylint issue is to do like this. 
https://docs.pytest.org/en/stable/reference/reference.html#pytest-fixture
https://itecnote.com/tecnote/python-pytest-fixtures-redefining-name-from-outer-scope-pylint/

Reply:
perfect!